package.json

```diff
@@ -18,7 +19,8 @@
     "lint": "npm run -s lintjs;npm run -s lintstyle",
     "fix": "npm run -s fixjs;npm run -s fixstyle",
     "-testing-": "--------testing--------",
-    "test": "./node_modules/jest-cli/bin/jest.js",
+    "test": "./node_modules/jest-cli/bin/jest.js --testPathIgnorePatterns='regtest'",
```

Comment:
I wanted to specify the ignore pattern here/inline, because if I put it lower down in the JSON it'll ignore the path for `yarn regtest` as well. One work-around would be to write another shell script that just fires off `jest regtest` and then never gets blocked by those ignore paths, but this makes a little more sense to me.

Reply:
awww dang you right  - sounds good! 👍